docs/document/docs/getting-started/how-to-use-kuc-types-in-typescript.md

```diff
@@ -52,24 +52,25 @@ npm install kintone-ui-component
 
 Modify ``src/index.ts``
 ```
-import { Dropdown, DropdownProps } from 'kintone-ui-component';
+import { Dropdown, DropdownProps, DropdownItem } from 'kintone-ui-component';
 
 const root = document.getElementById('root');
+const items: DropdownItem[] = [
```

Comment:
Not just DropdownItem?

After confirmation, I think we need to update the demo-app sample code under root, too.

Reply:
It's ok with DropdownItem[].